muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/schooldatalocal/LocalUserSchoolDataBridge.java

```diff
@@ -692,5 +692,4 @@ public boolean amICounselor(String studentIdentifier) {
   public List<UserStudyPeriod> listStudentStudyPeriods(SchoolDataIdentifier userIdentifier) {
     throw new SchoolDataBridgeInternalException("Not supported");
   }
-
```

Comment:
The whole file can be reset to HEAD if it's just linebreaks. Same with `UserSchoolDataBridge` and `PyramusUserSchoolDataBridge`